### PR TITLE
Adding check for icon in terminal tab, enabling tests for command

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Enable the tests for command. As the command result, the terminal tab icon is checked. 

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/17973

### How to test this PR?
Run devfile tests suite.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/e2e/pageobjects/ide/Terminal.ts [hunk 1]
```diff
@@ -134,6 +134,24 @@ export class Terminal {
...
+        let terminalClass = await this.driverHelper.waitAndGetElementAttribute(terminalTabLocator, 'class');
+
+        await this.driverHelper.getDriver().wait(async () => {
+            terminalClass = await this.driverHelper.waitAndGetElementAttribute(terminalTabLocator, 'class');
+            if (terminalClass.includes('fa-check')) { // css for tick icon
+                return true;
+            }
+            if (terminalClass.includes('fa-times-circle')) { // css for failed icon
+                Logger.error('The error occured when waiting for task "' + taskName + '" to succeed.');
+                throw new Error('The error occured when waiting for task "' + taskName + '" to succeed.');
```
Comment (rhopp) [inline tests/e2e/pageobjects/ide/Terminal.ts]: I think the error message here is quite general...
If I'm not mistaken, the error is thrown when the task fails (and *only* in this case). Thus I would go for something like 
```
throw new Error('Task "' + taskName + '" failed.');
```

File: tests/e2e/pageobjects/ide/Terminal.ts [hunk 2]
```diff
@@ -134,6 +134,24 @@ export class Terminal {
...
+            }
+            return false;
+        }, timeout);
```
Comment (rhopp) [inline tests/e2e/pageobjects/ide/Terminal.ts]: If you want, you can also specify error message for cases, when the `wait` times out for easier understanding of the logs if this case happens. Something like:
```suggestion
        }, timeout, "Timed out waiting for task " + taskName + " to succeed.");
        
```

**Review by rhopp — APPROVED** [review]
I've added two notes. It's up to you if you want to incorporate them into this PR.

File: tests/e2e/pageobjects/ide/Terminal.ts (hunk 1 above)
Comment (Katka92) [reply tests/e2e/pageobjects/ide/Terminal.ts]: Changed.

File: tests/e2e/pageobjects/ide/Terminal.ts (hunk 2 above)
Comment (Katka92) [reply tests/e2e/pageobjects/ide/Terminal.ts]: Good point, added.

**Comment by Katka92** [comment]
[crw-ci-test]

**Pull request merged**